jenkins/ompi/Dockerfile.centos

```diff
@@ -1,5 +1,5 @@
 ARG OMPI_CI_OS_NAME=centos
-ARG OMPI_CI_OS_VERSION=7
+ARG OMPI_CI_OS_VERSION=7.6.1810
```

Comment:
Could you please define `ARG OMPI_CI_MOFED_OS=rhel7.6` here and use it below? Next time we change it it will be easier to catch the mismatch.

Reply:
Fixed.